app/pages/SiloUtilizationPage.tsx

```diff
@@ -72,6 +74,12 @@ export function SiloUtilizationPage() {
     <>
       <PageHeader>
         <PageTitle icon={<Metrics24Icon />}>Capacity &amp; Utilization</PageTitle>
+        <DocsPopover
+          heading="Utilization"
+          icon={<Metrics16Icon />}
+          summary="System metrics let you monitor utilization of CPU, memory, and storage, in contrast to the total quota available to you."
```

Comment:
How about 

> System metrics let you monitor utilization of CPU, memory, and storage against silo-level quotas.

Reply:
Sorry, I meant that for system utilization, and for here I would leave out the bit about quotas:

> System metrics let you monitor utilization of CPU, memory, and storage